developers_guide_cn.md

```diff
@@ -54,9 +59,60 @@ Curve 社区有多个[沟通渠道](#社区交流)，每两周还会有线上的
 
 [Curve 测试用例编译及执行](https://github.com/opencurve/curve/blob/master/docs/cn/build_and_run.md#测试用例编译及执行)
 
+代码补全: 请使用 clangd 作为代码补全, 请见 [curve_clangd.md](https://github.com/opencurve/curve/blob/master/docs/cn/clangd.md).
+
+Curve CI 使用```cpplint```检查更改的代码,
+- 安装```cpplint``` (需要root):
+  ```bash
+  $ pip install cpplint
+  ```
+- 本地进行代码检查:
+  ```bash
+  $ cpplint --filter=-build/c++11 --quiet --recursive your_path
+  ```
+
+运行单元测试:
```

Comment:
<img width="548" alt="image" src="https://user-images.githubusercontent.com/15689619/216870608-1f7fa6df-428f-41dc-a4db-42eb0eff5097.png">

There is a file link on the compilation and operation of the single test. If you need to add it, it is recommended to add it in the above file.

Reply:
move it. good suggestions.